Makefile

```diff
@@ -36,10 +36,20 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time; see
+# https://github.com/golang/go/issues/13470.
+# If a static build is requested, assume musl is installed (it is in
+# the cockroachdb/builder docker container) and link against it
+# instead.
+CC = /usr/local/musl/bin/musl-gcc
```

Comment:
Are we going to continue to test the musl builds? If not, they will likely rot as all untested code does. I like Ben's suggestion to place the info about using musl for static builds on a wiki page along with some notes about the downsides to using musl (i.e. not tested).